ContractsApi/V1/Infrastructure/Exceptions/SuspendingBlockException.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace ContractsApi.V1.Infrastructure.Exceptions
+{
+    public class SuspendingBlockException : Exception
+    {
+        public SuspendingBlockException()
+            : base(string.Format("It is not possible to add a suspension to blocks"))
```

Comment:
is `string.Format` necessary here?

Reply:
Nope, it's a relic from bad copy paste. Deleted.